examples/tensorflow-profiling-examples/trainer-deep-profiling/task.py

```diff
@@ -0,0 +1,84 @@
+import argparse
+import json
+import os
+
+import model
+
+
+def parse_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--bucket',
+                        help='GCS path to data',
+                        required=True)
+    parser.add_argument('--output_dir',
+                        help='GCS loc to write checkpoints and export models',
+                        required=True)
+    parser.add_argument('--train_steps',
+                        help='Steps to run the training job for.',
+                        type=int,
+                        default=10000)
+    parser.add_argument('--eval_int',
+                        help='Evaluation interval in seconds',
+                        type=int,
+                        default=200)
+    parser.add_argument('--pattern',
+                        help='Specify a pattern that has to be in input files. For example 00001-of will process only one shard',  # pylint: disable=C0301
```

Comment:
Please wrap the text to not go over the 80 column limit

Reply:
ack